### PR TITLE
Add one 6 cup Chemex to things.yml

### PR DESCRIPTION
**Pull request by hakerja**

### CODE REVIEW

File: things.yml [hunk 1]
```diff
@@ -127,3 +127,7 @@ things:
     - number: 2
     - image: https://cdn.shopify.com/s/files/1/0748/6277/products/spring_loaded_glitter_bomb_multi_1200x.jpg?v=1512349319
     - link: https://www.ruindays.com/products/spring-loaded-glitter-bomb?variant=12317569155143
+  six-cup-chemex:
```
Comment (vsoch) [inline things.yml]: How pretty! Let's just call it `six-cup-coffeemaker` since us mortals don't know what a six cup chemex is!

Comment (vsoch) [reply things.yml]: The ?v is also probably some sort of visitor / tracking, so you can remove it from both links, for example:

https://cdn.shopify.com/s/files/1/0056/4562/products/chemex-at-home-web_large.jpg

**Review by vsoch — CHANGES_REQUESTED** [review]

File: things.yml (hunk 1 above)
Comment (hakerja) [reply things.yml]: thanks! new commit created with the suggested changes.

**Review by vsoch — APPROVED** [review]

**Comment by vsoch** [comment]
Looks good! Just think of all the coffee you could make... :coffee:

**Pull request merged**